cdap-app-templates/cdap-etl/cdap-etl-batch/src/test/java/co/cask/cdap/templates/etl/batch/ETLMapReduceTest.java

```diff
@@ -70,12 +70,26 @@ public void testConfig() throws Exception {
     }
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void testInvalidConfig() throws Exception {
```

Comment:
nit: there can be many types of invalid configs (for exaple, source name is missing from properties), better to be more specific